packages/firestore/test/integration/util/helpers.ts

```diff
@@ -17,53 +17,14 @@
 
 import * as firestore from '@firebase/firestore-types';
 import firebase from './firebase_export';
+import {
+  ALT_PROJECT_ID,
+  DEFAULT_PROJECT_ID,
+  DEFAULT_SETTINGS
+} from './settings';
 
 /* eslint-disable no-restricted-globals */
 
-/**
```

Comment:
Moved this code as importing this entire class breaks the Lite compile with typings conflicts.